driver/sdk.md

```diff
@@ -76,78 +84,269 @@ $ bblfsh-sdk update
 managed file "README.md" has changed, discarding changes
 ```
 
-## Creating the AST parser
+`bblfsh-sdk` doesn't update the SDK itself so if you want to use a new version of 
+the SDK you'll also have to update the Go package **first** with:
+
+```bash
+go get -u github.com/bblfsh/sdk/...
+```
+
+If the update gives you some problem you can try to delete the
+`$GOPATH/src/github.com/bblfsh/sdk` manually and run the go get command 
+again to get a fresh copy.
+
+## Implementing the Driver
+
+A driver has several subcomponents. On this document and the next one
+we'll see how to implement all these parts to create a new driver.
+
+The components are:
+
+- tests: source code, native AST, and UAST files that with for 
+  the integration tests.
+
+- Native parser: parses source code in the driver's target language
```

Comment:
The parser's language doesn't need to be the driver's target language, as stated below in the documentation.